opal/mca/pmix/pmix2x/pmix/src/mca/gds/ds12/gds_dstore.c

```diff
@@ -2804,6 +2808,16 @@ static pmix_status_t dstore_assign_module(pmix_info_t *info, size_t ninfo,
         }
     }
 
+<<<<<<< c632784ca34c467055eadcb4efe84a25e5a3911b:opal/mca/pmix/pmix2x/pmix/src/dstore/pmix_esh.c
```

Comment:
relax dude - do you not see the .pmix_ignore in that component? It is prevented from compiling due to the reported problems in the dstore. I am removing the ignore now as it seems functional, albeit the memory footprint problem remains